cluster/manifests/skipper/pod-deletion-cost-controller.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    application: skipper-ingress
+    component: pod-deletion-cost-controller
+  name: pod-deletion-cost-controller
+  namespace: kube-system
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      deployment: pod-deletion-cost-controller
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
```

Comment:
I think [Recreate strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#recreate-deployment) fits better to ensure single instance like we use for fabric-gateway-controller

Reply:
It should not matter and I do not think that "ensure single" is happening anyways, regardless of the config. 
Think of it more that "most often" it will be like you expect.
I will change as you suggest.

Reply:
done